runtimes/peregrine/src/governance.rs

```diff
@@ -0,0 +1,292 @@
+				Some(AccountId::new([0; 32]))
+			} else {
+				<()>::get_prime()
+			}
+		}
+	}
+}
+
+type TipsMembershipProvider = pallet_membership::Instance2;
+impl pallet_membership::Config<TipsMembershipProvider> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type AddOrigin = RootOrMoreThanHalfCouncil;
+	type RemoveOrigin = RootOrMoreThanHalfCouncil;
+	type SwapOrigin = RootOrMoreThanHalfCouncil;
+	type ResetOrigin = RootOrMoreThanHalfCouncil;
+	type PrimeOrigin = RootOrMoreThanHalfCouncil;
+	type MembershipInitialized = ();
+	type MembershipChanged = MockMembershipChangedForBenchmarks;
+	type MaxMembers = constants::governance::TipperMaxMembers;
+	type WeightInfo = weights::pallet_membership::WeightInfo<Runtime>;
+}
+
+parameter_types! {
+	pub const PreImageHoldReason: RuntimeHoldReason = RuntimeHoldReason::Preimage(pallet_preimage::HoldReason::Preimage);
+}
+
+impl pallet_preimage::Config for Runtime {
+	type WeightInfo = weights::pallet_preimage::WeightInfo<Runtime>;
+	type RuntimeEvent = RuntimeEvent;
+	type Currency = Balances;
+	type ManagerOrigin = EnsureRoot<AccountId>;
+	type Consideration = HoldConsideration<
+		AccountId,
+		Balances,
+		PreImageHoldReason,
+		LinearStoragePrice<constants::preimage::PreimageBaseDeposit, constants::ByteDeposit, Balance>,
+	>;
+}
+
+#[allow(clippy::arithmetic_side_effects)]
+#[inline]
+fn maximum_scheduler_weight() -> Weight {
+	Perbill::from_percent(80) * BlockWeights::get().max_block
+}
+
+parameter_types! {
+	pub MaximumSchedulerWeight: Weight = maximum_scheduler_weight();
+}
+
+/// Used the compare the privilege of an origin inside the scheduler.
+pub struct OriginPrivilegeCmp;
+
+impl PrivilegeCmp<OriginCaller> for OriginPrivilegeCmp {
+	fn cmp_privilege(left: &OriginCaller, right: &OriginCaller) -> Option<Ordering> {
+		if left == right {
+			return Some(Ordering::Equal);
+		}
+
+		match (left, right) {
+			// Root is greater than anything.
+			(OriginCaller::system(frame_system::RawOrigin::Root), _) => Some(Ordering::Greater),
+			// Check which one has more yes votes.
+			(
+				OriginCaller::Council(pallet_collective::RawOrigin::Members(l_yes_votes, l_count)),
+				OriginCaller::Council(pallet_collective::RawOrigin::Members(r_yes_votes, r_count)),
+			) => Some((l_yes_votes.saturating_mul(*r_count)).cmp(&(r_yes_votes.saturating_mul(*l_count)))),
+			// For every other origin we don't care, as they are not used for `ScheduleOrigin`.
+			_ => None,
+		}
+	}
+}
+
+impl pallet_scheduler::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type RuntimeOrigin = RuntimeOrigin;
+	type PalletsOrigin = OriginCaller;
+	type RuntimeCall = RuntimeCall;
+	type MaximumWeight = MaximumSchedulerWeight;
+	type ScheduleOrigin = RootOrCollectiveProportion<CouncilCollective, 1, 2>;
+	type MaxScheduledPerBlock = ConstU32<50>;
+	type WeightInfo = weights::pallet_scheduler::WeightInfo<Runtime>;
+	type OriginPrivilegeCmp = OriginPrivilegeCmp;
+	type Preimages = Preimage;
+}
```

Comment:
Is this not more a system pallet?

Reply:
Moved it into the system module: https://github.com/KILTprotocol/kilt-node/pull/797/commits/bf159368bb2127655073a6f5e5849f1fe6781285.